calicoctl/resourcemgr/hostendpoint.go

```diff
@@ -15,15 +15,20 @@
 package resourcemgr
 
 import (
-	"github.com/projectcalico/libcalico-go/lib/api"
-	"github.com/projectcalico/libcalico-go/lib/api/unversioned"
-	"github.com/projectcalico/libcalico-go/lib/client"
+	"context"
+
+	api "github.com/projectcalico/libcalico-go/lib/apiv2"
+	client "github.com/projectcalico/libcalico-go/lib/clientv2"
+	"github.com/projectcalico/libcalico-go/lib/options"
 )
 
 func init() {
+	ctx := context.Background()
 	registerResource(
 		api.NewHostEndpoint(),
 		api.NewHostEndpointList(),
+		false,
+		[]string{"hostendpoint", "hostendpoints", "hep", "heps"},
 		[]string{"NODE", "NAME"},
```

Comment:
I think the NAME field should be the leading field for each resource now.

I kind feel we should remove name from these lists and add it in automagically (basically just include the Spec fields).  WDYT?

Reply:
> I think the NAME field should be the leading field for each resource now.

I agree

> I kind feel we should remove name from these lists and add it in automagically (basically just include the Spec fields). WDYT?

Not sure I understand what you mean by adding the name field automagically. Can you give me an example?